docker/core/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin sh
+sh /home/sgx_config.sh
+node /home/ehsm/ehsm_kms_service/ehsm_kms_server.js
```

Comment:
combine the twi scripts into one, (named as start_ehsm-kms.sh)

Reply:
It will be fixed